specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/payments.json

```diff
@@ -0,0 +1,572 @@
+{
+  "consumes": [
+    "application/json"
+  ],
+  "definitions": {
```

Comment:
Just for consistancy with the rest of the repositories, do you mind ordering your top level properties
- info
- paths
- definitions
- parameters
- etc...

Reply:
done